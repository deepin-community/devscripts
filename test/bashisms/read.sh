#!/bin/sh

read
read -r
read file.txt

read-local-command
read-local-command -r
read-local-command file.txt
