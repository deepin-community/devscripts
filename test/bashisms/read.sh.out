possible bashism in bashisms/read.sh line 3 (read without variable):
read
possible bashism in bashisms/read.sh line 4 (read without variable):
read -r
